wasp/src/main/java/com/orhanobut/wasp/VolleyNetworkStack.java

```diff
@@ -114,25 +117,28 @@ public void onResponse(T response) {
         @Override
         public void onErrorResponse(VolleyError error) {
             int statusCode = WaspError.INVALID_STATUS_CODE;
-            byte[] body = new byte[0];
+            String body = "";
+            int length = 0;
             Map<String, String> headers = new HashMap<>();
             long delay = 0;
             if (error == null) {
-                callBack.onError(new WaspError(url, statusCode, headers, "No message", body, delay));
+                callBack.onError(new WaspError(
```

Comment:
following format would be more consistent. 
<pre>
callBack.onError(new WaspError(
           parser, new WaspResponse(url, statusCode, headers, body, length, delay), "No message"
));
</pre>